src/components/pages/get-help/cards/cards.jsx

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Button from 'components/shared/button';
+import Container from 'components/shared/container';
+import Heading from 'components/shared/heading';
+
+import GithubIcon from './images/github.inline.svg';
+import SlackIcon from './images/slack.inline.svg';
+import SupportIcon from './images/support.inline.svg';
+
+const icons = {
+  github: GithubIcon,
+  support: SupportIcon,
+  slack: SlackIcon,
+};
+
+const Cards = ({ items }) => (
+  <section className="bg-gray-4 pt-12 pb-28">
```

Comment:
I'm afraid we can not use `section` tag here due to absent heading tag as mentioned [here](https://stackoverflow.com/questions/8544671/is-it-necessary-to-have-a-heading-of-section-in-html5), let's switch to div